packages/flutter_tools/test/general.shard/runner/flutter_command_runner_test.dart

```diff
@@ -176,24 +175,6 @@ void main() {
         Analytics: () => fakeAnalytics,
       });
 
-      group('${FlutterGlobalOptions.kPrintDtd} flag', () {
```

Comment:
where is the `--print-dtd` behavior test covered by flutter tools now?

Reply:
There wasn't an obvious way to test this after moving it out of a global variable and into `DebuggingOptions`. I've reverted the change for this PR and added back the test, but will investigate options for testing in a follow up PR when I remove the global flag.